packages/extension-polkagate/src/fullscreen/settings/addressBook/quickAddContact/AddToAddressBook.tsx

```
@@ -0,0 +1,101 @@
+
+    useEffect(() => {
+        // We include `addingContact` to prevent the component from being removed
+        // immediately after the address is added to the address book.
+        // This allows the UI transition (fade-out) to complete smoothly.
+        if (!substrate || !contacts || addingContact) {
+            return;
+        }
+
+        const existingAddresses = new Set<string>();
+
+        accounts.forEach(({ address }) => existingAddresses.add(address));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)